### PR TITLE
[Button v1] Update styles and documentation

### PR DESCRIPTION
**Pull request by chenyejia**
* Tertiary, small, and disabled buttons
* Added documentation to `Button` component
* Misc clean up

### CODE REVIEW

**Comment by jeresig** [comment]
Deploy preview for *wonder-blocks* ready!

Built with commit cdedc32e18d4b9963cef18b44dbf28657c822fe3

https://deploy-preview-92--wonder-blocks.netlify.com

File: packages/wonder-blocks-button/docs.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 **WARNING: Button is not finished, do not use!**
```
Comment (jeresig) [inline packages/wonder-blocks-button/docs.md]: Probably safe to remove this now! 😄

File: packages/wonder-blocks-button/components/button-core.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ export default class ButtonCore extends React.Component<Props> {
         ];
 
         const Tag = href ? StyledAnchor : StyledButton;
-
+        const Label = size === "small" ? LabelSmall : LabelLarge;
```
Comment (kevinbarabash) [inline packages/wonder-blocks-button/components/button-core.js]: nice

File: packages/wonder-blocks-button/components/button-core.js [hunk 3]
```diff
@@ -184,19 +189,23 @@ const _generateStyles = (color, kind, light) => {
...
-                borderStyle: "solid",
-                borderWidth: 2,
-                paddingLeft: 2,
-                paddingRight: 2,
+                ":after": {
+                    content: "''",
+                    position: "absolute",
+                    height: 2,
+                    width: "calc(100% - 8px)",
+                    bottom: "calc(50% - 11px)",
```
Comment (kevinbarabash) [inline packages/wonder-blocks-button/components/button-core.js]: sneaky

**Pull request merged**